apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerClientBuilderPartial.cs

```diff
@@ -76,5 +87,17 @@ public SpannerClientBuilder MaybeCreateEmulatorClientBuilder()
             builder.CopySettingsForEmulator(this);
             return builder;
         }
+
+        internal new T GetEffectiveSettings<T>(T settings) where T : ServiceSettingsBase, new()
```

Comment:
Humbug - I guess we should have made the method virtual in GAX? Not that it's a problem, so long as we don't call it *from* GAX.